pkg/util/docker/global.go

```diff
@@ -211,20 +211,41 @@ func ResolveImageName(image string) (string, error) {
 	if globalDockerUtil != nil {
 		return globalDockerUtil.extractImageName(image), nil
 	}
-	return "", errors.New("dockerutil not initialised")
+	return image, errors.New("dockerutil not initialised")
 }
 
-// SplitImageName splits a valid image name (from ResolveImageName)
-// into the name and tag parts.
-func SplitImageName(image string) (string, string, error) {
+// SplitImageName splits a valid image name (from ResolveImageName) and returns:
+//    - the "long image name" with registry and prefix, without tag
+//    - the "short image name", without registry, prefix nor tag
+//    - the image tag if present
+//    - an error if parsing failed
+func SplitImageName(image string) (string, string, string, error) {
```

Comment:
Could we use a struct here instead of 3 strings? `ImageNameFragments.LongName|ShortName|Tag`
I'm pretty sure this will evolve again later and updating all the calls to fit a small signature change is busy work.